_includes/component.html

```diff
@@ -13,19 +15,29 @@ <h2 class="usa-alert__heading">{{ page.alert-heading }}</h2>
 </div>
 {% endif %}
 
+<!--Add wcag compliance status tag-->
```

Comment:
```suggestion
<!-- Add wcag compliance status tag -->
```

Reply:
Fixed in [72386ab](https://github.com/uswds/uswds-site/pull/2138/commits/72386ab6f0ba9f762be1c9f7475a881628d37cb7)